src/Illuminate/Http/ApiResponseTrait.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Illuminate\Http;
+
+use InvalidArgumentException;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Pagination\AbstractPaginator;
+
+trait ApiResponseTrait
```

Comment:
This trait actually describes what the class _is_, so `IsApiResponse` sounds good to me.